### PR TITLE
Gives ranged attack procs for superior_animal

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
title

## Why It's Good For The Game
superior animals will allow you to get shot in maint

## Changelog
:cl:
code: steals ranged attacks from simple_mob
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/superior_animal/attack.dm [hunk 1]
```diff
@@ -10,4 +10,66 @@
...
+				else
+					OpenFire(target_mob)
+			else
+				if(prob(45))
+					stance = HOSTILE_STANCE_ATTACKING
+					set_glide_size(DELAY2GLIDESIZE(move_to_delay))
+					walk_to(src, target_mob, 1, move_to_delay)
+				else
+					OpenFire(target_mob)
+		else
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/superior_animal/attack.dm:39]: Else?

File: code/modules/mob/living/carbon/superior_animal/attack.dm [hunk 2]
```diff
@@ -10,4 +10,66 @@
...
+	
+/mob/living/carbon/superior_animal/proc/OpenFire(target_mob)
+	var/target = target_mob
+	visible_message("\red <b>[src]</b> [fire_verb] at [target]!", 1)
```
Comment (TheShown911) [inline code/modules/mob/living/carbon/superior_animal/attack.dm:43]: Would probably use a span macro like span danger.

**Review by TheShown911 — COMMENTED** [review]
There's no way to trigger these procs yet, right?

**Comment by zombiemimic** [comment]
> There's no way to trigger these procs yet, right?

Not yet, no

**Comment by vode-code** [comment]
> 
> 
> > There's no way to trigger these procs yet, right?
> 
> Not yet, no

if this gets merged, I'll make Gestralhte roaches use this.

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**